CONTRIBUTING.md

```diff
@@ -2,59 +2,64 @@
 Welcome! Thanks for checking out Expensify.cash and for taking the time to contribute!
 
 ## Getting Started
-This guide is specifically for external contributors. For a general overview of the repo, check out our README located [here](https://github.com/Expensify/Expensify.cash/blob/master/README.md). The part of the README to pay particular attention to is how to [run the app](https://github.com/Expensify/Expensify.cash#local-development) locally using our production API.
+This guide is for external contributors. Check out our README guidelines here for a general overview of the code repository, how to run the app locally, testing, storage, etc. 
```

Comment:
>This guide is for external contributors.

NAB, this is sort of implied since it's a `CONTRIBUTING.md`

Reply:
Updated the phrasing to be more accurate. Hopefully it's better now?